txp/Pos/Txp/Toil/Failure.hs

```diff
@@ -39,7 +41,7 @@ data ToilVerFailure
     | ToilOutGreaterThanIn { tInputSum  :: !Integer
                            , tOutputSum :: !Integer}
     | ToilInconsistentTxAux !Text
-    | ToilInvalidOutputs !Text  -- [CSL-1628] TODO: make it more informative
+    | ToilInvalidOutputs !Word32 !TxOutVerFailure
```

Comment:
Probably it makes sense to rename it to `ToilInvalidOutput`, because only one output is invalid in this case :)